### DISCUSSION TITLE
Max GPU TDP differs depending on the CPU mode used.

### DISCUSSION THREAD

**Discussion by Kikimahe** [seerge/g-helper#2466] [Q&A] [answered]
Hi,

With Furmak and HWinfo64, I noticed that the GPU TDP was different depending on the CPU mode used.

My graphics card has a maximum GPU TDP of 140W (RTX 3060)
- CPU Silent, the GPU is limited to 115W max
- CPU balanced the GPU is limited to 125W max
- CPU Turbo the GPU reaches its maximum power of 140W.

Is this something normal for all laptops or is it just on mine?

PS: I absolutely don't want to install Armory Crate to see if it's the same thing. ;-)

**Comment by IceStormNG** [comment, marked as answer]
This is in hardware and controlled by the EC. Pretty much all ASUS ROG Laptops have this, as they reduce the power based on cooling. Silent and "Balanced" have lower fan RPMs and therefore less cooling, so it makes no sense to let the GPU run at max wattage if the system is not able to cool it.

**Reply by Kikimahe** [reply]
Merci pour votre réponse. 👍

**Reply by chockokruasan** [reply]
Yes, the CPU and GPU TDP is different depending on the mode used, but G-helper changes my GPU TDP to the max on all modes, Is it only on my model? :)
video https://youtu.be/CUuj-Xoj0lY

**Reply by Kikimahe** [reply]
I continue in #2456  ;-)
